infra-test/guidance.md

```diff
@@ -0,0 +1,12 @@
+# Guidance to the candidate
+
+1. We want to make sure that you are a good fit to the culture and the work ethic of the team and company.
+1. We ask that it is returned within 3 working days of receiving the test if possible but please let us know if you require additional time. 
+1. The test is not time restricted - we do not force a maximum time limit to complete it - though it is suggested to be completed in approximately 2 hours. There is also no minimum time restriction.
```

Comment:
The test is not time restricted - you can spend as much, or as little, time as you like on it as long as you return it within 3 days.

Reply:
yes exactly.